src/views/Zniffer.vue

```diff
@@ -1025,6 +1042,68 @@ export default {
 				)
 			}
 		},
+		loadCapture() {
```

Comment:
Instead of using an hidden input I have an util method on App.vue that you can call using this.app.importFile, check this has instancemixins (so this.app is defined)